### PR TITLE
Audisp plugin config path implied property

### PR DESCRIPTION
**Pull request by freddieRv**
#### Description:

- Add an implied property for the config path for audisp plugin

#### Rationale:

- Config options for audisp plugin are placed in /etc/audisp/auditd.conf for audit package previous to v3
- I based this work on https://github.com/ComplianceAsCode/content/pull/6986
- This is an alternative fix for the issue described on https://github.com/ComplianceAsCode/content/pull/7573

### CODE REVIEW

**Comment by freddieRv** [comment]
Marking as WIP because I would like some feedback on the following:

As the logic is right now, with the jinja macros, the default config path is /etc/audisp/, except for "rhel8", "fedora", "ol8", and "rhv4".

While this update makes the default /etc/audit/. This requires the implied property to be overrided in every other product that is not in "rhel8", "fedora", "ol8", or "rhv4".

**Review by jan-cerny — CHANGES_REQUESTED** [review]
please fix the OVAL but otherwise lgtm

File: linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/oval/shared.xml [hunk 1]
```diff
@@ -1,9 +1,5 @@
-{{% if product in ["rhel8", "fedora", "ol8", "rhv4"] %}}
-{{% set audisp_config_file_path = "/etc/audit/audisp-remote.conf" %}}
-{{% else %}}
-{{% set audisp_config_file_path = "/etc/audisp/audisp-remote.conf" %}}
+{{% set audisp_config_file_path = "{{{ audisp_conf_path }}}/audisp-remote.conf" %}}
```
[the remark this answers is not in the document]
Comment (freddieRv) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/oval/shared.xml]: Addressed. Thanks for the feedback!

**Review by jan-cerny — APPROVED** [review]
```
[jcerny@thinkpad scap-security-guide{pr/7594}]$ python3 tests/test_suite.py  rule   --libvirt qemu:///system ssgts_rhel8 auditd_audispd_encrypt_sent_records,auditd_audispd_syslog_plugin_activated,auditd_audispd_configure_remote_server,auditd_overflow_action
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-06-15-0851/test_suite.log
WARNING - Script audisp_remote_server_hostname.pass.sh is not applicable on given platform
WARNING - Script audisp_remote_server_not_there.fail.sh is not applicable on given platform
WARNING - Script encrypt_sent_records.pass.sh is not applicable on given platform
WARNING - Script encrypt_sent_records_not_activated.fail.sh is not applicable on given platform
WARNING - Script encrypt_sent_records_not_there.fail.sh is not applicable on given platform
WARNING - Script audisp_syslog_plugin_activated.pass.sh is not applicable on given platform
WARNING - Script audisp_syslog_plugin_activated_not_there.fail.sh is not applicable on given platform
WARNING - Script audisp_syslog_plugin_not_activated.fail.sh is not applicable on given platform
INFO - xccdf_org.ssgproject.content_rule_auditd_audispd_configure_remote_server
INFO - Script audit_remote_server_hostname.pass.sh using profile (all) OK
INFO - Script audit_remote_server_not_there.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_auditd_audispd_encrypt_sent_records
INFO - Script transport_bogus_value.fail.sh using profile (all) OK
INFO - Script transport_correct_value.pass.sh using profile (all) OK
INFO - Script transport_not_there.fail.sh using profile (all) OK
INFO - Script transport_wrong_value.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_auditd_audispd_syslog_plugin_activated
INFO - Script audit_syslog_plugin_activated.pass.sh using profile (all) OK
INFO - Script audit_syslog_plugin_activated_not_there.fail.sh using profile (all) OK
INFO - Script audit_syslog_plugin_not_activated.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_auditd_overflow_action
INFO - Script commented_out.fail.sh using profile (all) OK
INFO - Script empty.fail.sh using profile (all) OK
INFO - Script file_not_present.fail.sh using profile (all) OK
INFO - Script halt.pass.sh using profile (all) OK
INFO - Script ignore.fail.sh using profile (all) OK
INFO - Script not_present.fail.sh using profile (all) OK
INFO - Script single.pass.sh using profile (all) OK
INFO - Script syslog.pass.sh using profile (all) OK
[jcerny@thinkpad scap-security-guide{pr/7594}]$ 
```

**Pull request merged**